Gruntfile.js

```diff
@@ -102,12 +101,36 @@ module.exports = function (grunt) {
 
   grunt.registerTask('build', ['checkGitSubmodules', 'webpack:all', 'build:browser']);
 
-  grunt.registerTask('build:node', ['checkGitSubmodules', 'webpack:node']);
-
   grunt.registerTask('build:browser', ['checkGitSubmodules', 'webpack:browser']);
 
   grunt.registerTask('all', ['build', 'requirejs']);
 
+  grunt.registerTask('build:node', function () {
+    const done = this.async();
+
+    const nodeConfig = {
```

Comment:
Is it possible to DRY up this and the stuff in `build:browser`, the same way as `webpack.config.js` has a `baseConfig` etc?

Reply:
good idea, will do!